platform/ABModel.js

```diff
@@ -1331,6 +1333,12 @@ module.exports = class ABModel extends ABModelCore {
 
       // special operation cases:
       switch (condition.rule) {
+         case "like":
+            // like: "searchTermWith%"
+            operator = "LIKE";
+            value = quoteMe(condition.value);
```

Comment:
Do we need to have `%` ?

Reply:
well, I thought that if we are allowing someone to put in `like` as a condition, then they could choose where the `%` operator would go:
- search`%`
- `%`search
- `%`search`%`

otherwise they could use one of our other conditions like `begins_with`, `ends_with`, `contains`,  etc...